backend/src/main/resources/db/migration/V2023.01.10.1148__create_favorite_activity.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE spar.favorite_activity (
```

Comment:
I would highly suggest to always use `IF NOT EXISTS` to every SQL statement to prevent issues during migration.